resources/lib/services/nfsession/session/base.py

```diff
@@ -44,7 +44,7 @@ def _init_session(self):
         import httpx
         # (http1=False, http2=True) means that the client know that server support HTTP/2 and avoid to do negotiations,
         # prior knowledge: https://python-hyper.org/projects/hyper-h2/en/v2.3.1/negotiating-http2.html#prior-knowledge
-        self.session = httpx.Client(http1=False, http2=True)
+        self.session = httpx.Client(http1=False, http2=True, follow_redirects=True)
```

Comment:
`follow_redirects` on this httpx version is already set as true

Reply:
But the default changes depending on the httpx version, as far as I understood. With this patch the code works in any case.

Reply:
you cant change follow_redirects in the old version so add this is useless